### PR TITLE
[skip ci]Speed up the cluster test, manually got it to 21 minutes

### PR DESCRIPTION
**Pull request by mhagen-vmware**
#5386

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot [hunk 1]
```diff
@@ -20,17 +20,29 @@ Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
...
+    Should Contain  ${output.stdout}  Overall Status: Succeeded
+    ${output}=  Split To Lines  ${output.stdout}
+
+    Open Connection  %{NIMBUS_GW}
+    Wait Until Keyword Succeeds  2 min  30 sec  Login  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}
+    ${vc-ip}=  Get IP  ${vc}
+    Close Connection
+
+    Set Environment Variable  GOVC_INSECURE  1
+    Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot]: Should these credentials be hardcoded here?

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot]: Yep, they don't change.  They are already hard coded in the other deployment keywords as well.

Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot]: Just for my knowledge: we weren't setting these variables previously. Is there a reason we're doing this now?

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot]: It was previously set in the deploy vcenter server keyword before.

Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-2-Cluster.robot]: Thanks!

**Pull request merged**